Tasks/CondaEnvironmentV1/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 140,
```

Comment:
Is it okay for the version numbers to all be the same in these releases? I assume it is, but just wanted to ask.

Reply:
yes -- our policy for changes we are hotfixing has been to use the oldest milestone we are hotfixing to.  When we release the hotfix, only one package will be published (1.140.2) and that will get picked up in releases/m140 and releases/m141 on the service.